### PR TITLE
Advertise velocity multiplexer services to change topic priorities

### PR DESCRIPTION
**Pull request by SomeshDaga**
The following implements the services defined in https://github.com/akrobotics/yocs_msgs/pull/1

### CODE REVIEW

File: yocs_cmd_vel_mux/src/cmd_vel_mux_nodelet.cpp [hunk 1]
```diff
@@ -27,6 +29,80 @@ namespace yocs_cmd_vel_mux {
  ** Implementation
  *****************************************************************************/
 
+bool CmdVelMuxNodelet::changeTopicPriorityServiceCb(yocs_msgs::ChangeMuxPriority::Request& req,
```
Comment (LPitonakova) [inline yocs_cmd_vel_mux/src/cmd_vel_mux_nodelet.cpp:32]: Please provide a brief and a description of parameters and return value on top the 2 new functions.

File: yocs_cmd_vel_mux/src/cmd_vel_mux_nodelet.cpp [hunk 2]
```diff
@@ -59,6 +135,12 @@ void CmdVelMuxNodelet::cmdVelCallback(const geometry_msgs::Twist::ConstPtr& msg,
   }
 }
 
+void CmdVelMuxNodelet::timerCallback(const ros::TimerEvent& event, unsigned int idx, uint8_t priority)
```
Comment (LPitonakova) [inline yocs_cmd_vel_mux/src/cmd_vel_mux_nodelet.cpp:138]: Same comment as above here as well, and may be worth doing for other functions that you looked at but didn't change - I'd suggest we try to make this as readable for others as possible.

**Review by LPitonakova — COMMENTED** [review]
Please see comments in code

File: yocs_cmd_vel_mux/src/cmd_vel_mux_nodelet.cpp (hunk 2 above)
Comment (SomeshDaga) [reply yocs_cmd_vel_mux/src/cmd_vel_mux_nodelet.cpp:138]: Addressed

File: yocs_cmd_vel_mux/src/cmd_vel_mux_nodelet.cpp (hunk 1 above)
Comment (SomeshDaga) [reply yocs_cmd_vel_mux/src/cmd_vel_mux_nodelet.cpp:32]: Addressed

Comment (SomeshDaga) [reply yocs_cmd_vel_mux/src/cmd_vel_mux_nodelet.cpp:32]: Done. Please refer to the header file

File: yocs_cmd_vel_mux/src/cmd_vel_mux_nodelet.cpp (hunk 2 above)
Comment (SomeshDaga) [reply yocs_cmd_vel_mux/src/cmd_vel_mux_nodelet.cpp:138]: Done. Refer to header file

**Review by LPitonakova — APPROVED** [review]

**Review by raadkhan — APPROVED** [review]
LGTM :+1:

**Pull request merged**